mlflow/java/scoring/src/main/java/org/mlflow/LoaderModule.java

```diff
@@ -51,7 +51,7 @@ public Predictor load(String modelRootPath) throws PredictorLoadingException {
     } catch (IOException e) {
       throw new PredictorLoadingException(
           "Failed to load the model configuration at the specified path. Please ensure that"
-              + " this is the path to the root directory of a valid MLflow model");
+              + " this is the path to the root directory of a valid MLflow model", e);
```

Comment:
Extended `PredictorLoadingException` to include the root cause exception for easier debugging next time